src/diffpy/labpdfproc/labpdfprocapp.py

```diff
@@ -36,7 +37,7 @@ def get_args():
     p.add_argument(
         "-x",
         "--xtype",
-        help=f"the quantity on the independnt variable axis. allowed "
+        help=f"the quantity on the independent variable axis. allowed "
         f"values: {*XQUANTITIES, }. If not specified then two-theta "
         f"is assumed for the independent variable. Only implemented for "
```

Comment:
please check, do we have issues for implementing for other x-axes? if not, please could you make that issue?

Reply:
just added this